recipes/mob_suite/meta.yaml

```diff
@@ -16,11 +16,11 @@ source:
 
 requirements:
   host:
-    - python
+    - python >=3.4
```

Comment:
you already have `skip: True  # [not py36]` above ... are you sure this solves your problem?